### ISSUE TITLE
New item: FindText

### ISSUE THREAD

**Issue by hoppfrosch** [ahkscript/awesome-AutoHotkey#101]
### Item to be added:
FindText

### Author of the new item
feiyue

### Awesome-AHK chapter the item should be added to 
???

### Description of the new Item
Catch screen image into text and then find it

### Discussion Links:
Forum: https://autohotkey.com/boards/viewtopic.php?f=6&t=17834

### Source Links: 
Source: https://autohotkey.com/boards/viewtopic.php?f=6&t=17834

**Comment by hi5** [comment]
I must admit I have no clue what this is supposed to do and I read the thread three times. :confused:

**Comment by ed1chandler** [comment]
I just posted to the forum with a detailed walk-through of how to use it.  I'm just not following what a couple of the parameters do.

**Comment by hoppfrosch** [comment]
As nobody voted yet, I've marked it as a candidate for the sack. I'll leave it in this state for a few weeks - if there's no reaction, I'll consider this item as rejected

**Comment by joedf** [comment]
It seems to be OCR software, with a part in mcode. If this is to be added it would be under scripts -> ?
Math, Graphics? or its own category "Text"?

**Comment by hi5** [comment]
I think I understand it a bit better now, so could be included even though the documentation could be better people seem to find it useful. You can define an image to look for "in text" so you don't need an image for your ImageSearch but a block of text. (something like that)

**Comment by joedf** [comment]
Yeah... It is a little bit confusing given how it's being presented.

**Comment by hi5** [comment]
Similar #130
